nipype/interfaces/ants/resampling.py

```diff
@@ -246,9 +246,13 @@ class ApplyTransformsInputSpec(ANTSCommandInputSpec):
     interpolation_parameters = traits.Either(traits.Tuple(traits.Int()),  # BSpline (order)
                                              traits.Tuple(traits.Float(),  # Gaussian/MultiLabel (sigma, alpha)
                                                           traits.Float())
-                                             )
-    transforms = InputMultiPath(
-        File(exists=True), argstr='%s', mandatory=True, desc='transform files: will be applied in reverse order. For example, the last specified transform will be applied first')
+                                            )
+    transforms = traits.Either(InputMultiPath(File(exists=True), desc='transform files: will be '
```

Comment:
@blakedewey I think it is equivalent to what I did.
@oesteban I designed it to be backwards compatible. Did I miss something? I figured it is a change anyone solving a similar problem would appreciate.